File: app_canibuythat/src/main/java/com/gb/canibuythat/util/Logger.java
package com.gb.canibuythat.util;

import android.util.Log;

public class Logger {

    private static final String TAG = "CanIBuyThat";

    public static void d(String tag, String message) {
        Log.d(TAG, "(" + tag + ") " + message);
    }

    public static void d(String tag, Throwable t) {
        Log.d(TAG, "(" + tag + ")", t);
    }

    public static void e(String tag, String message) {
        Log.e(TAG, "(" + tag + ") " + message);
    }

    public static void e(String tag, Throwable t) {
        Log.e(TAG, "(" + tag + ")", t);
    }

    public static void v(String tag, String message) {
        Log.v(TAG, "(" + tag + ") " + message);
    }
}

File: app_canibuythat/src/main/java/com/gb/canibuythat/util/Logger.kt
package com.gb.canibuythat.util

import android.util.Log

object Logger {

    private val TAG = "CanIBuyThat"

    fun d(tag: String, message: String) {
        Log.d(TAG, "($tag) $message")
    }

    fun d(tag: String, t: Throwable) {
        Log.d(TAG, "($tag)", t)
    }

    fun e(tag: String, message: String) {
        Log.e(TAG, "($tag) $message")
    }

    fun e(tag: String, message: String, cause: Throwable) {
        Log.e(TAG, "($tag) $message", cause)
    }

    fun e(tag: String, t: Throwable) {
        Log.e(TAG, "($tag)", t)
    }

    fun v(tag: String, message: String) {
        Log.v(TAG, "($tag) $message")
    }
}
